libs/lambda-edge-openid-auth/src/lib/handlers/redirect.ts

```diff
@@ -16,7 +16,7 @@ export function redirect(
         redirect_uri: config.redirectUri,
         response_type: 'code',
         response_mode: 'query',
-        scope: 'openid offline_access',
+        scope: 'openid offline_access email profile',
```

Comment:
Instead of hardcoding the scopes, it's probably better to allow it to be set using config. See [libs/lambda-edge-openid-auth/src/lib/config.ts](https://github.com/sevenwestmedia-labs/lambda-edge-openid-auth/blob/6cd487428d95702fc9f8682a72632b77104aab3f/libs/lambda-edge-openid-auth/src/lib/config.ts#L39-L72) for details

The current value (`openid offline_access`) is a good default

Reply:
Great idea, thanks! I _think_ that I have implemented what you mean :P

Reply:
_continued in https://github.com/sevenwestmedia-labs/lambda-edge-openid-auth/pull/12#discussion_r1719197848_